src/EditorFeatures/CSharpTest2/Recommendations/WhenKeywordRecommenderTests.cs

```diff
@@ -425,19 +425,46 @@ class C
     void M() { switch (new object()) { case var $$ when } }
 }");
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/30848"), Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
```

Comment:
This is not new behavior, it worked before the tests were skipped. Just adding an additional test case.